src/app/pages/ArticlePage/ArticlePage.jsx

```diff
@@ -110,6 +114,11 @@ const ArticlePage = ({ pageData }) => {
     mostRead: mostReadInitialData,
   } = pageData;
 
+  const atiData = {
+    ...atiAnalytics,
+    ...(isPGL && { pageTitle: `${atiAnalytics.pageTitle} - ${brandName}` }),
```

Comment:
Do we need to add a unit test for this, or is that overkill?

Reply:
Yeah we should actually. Maybe a basic snapshot test to cover all bases? Or does that go against our code standards

Reply:
Yeah I think we're trying to get away from snapshots if we can. If there was a test where you could just check that the pageTitle is as expected when the page is a PGL, that would suffice

Reply:
Done, took a lil bit to jog my memory on mocking, I've also added missing test coverage for the secondary column and most read components